Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -326,21 +327,78 @@ PhysicalParticleContainer::AddPlasmaFromFile(const std::string s_f, amrex::Real
                                      "should contain only one species\n");
     std::pair<std::string,openPMD::ParticleSpecies> ps = *i.particles.begin();
 
-    //TODO: In future PRs will add AMREX_ALWAYS_ASSERT_WITH_MESSAGE to test if mass and charge are both const
+    //TODO: In future PRs will add ASSERT_WITH_MESSAGE to test if mass and charge are both const
     amrex::Real p_m = ps.second["mass"][openPMD::RecordComponent::SCALAR].loadChunk<amrex::Real>().get()[0];
     amrex::Real p_q = ps.second["charge"][openPMD::RecordComponent::SCALAR].loadChunk<amrex::Real>().get()[0];
-    int npart = ps.second["position"]["x"].getExtent()[0];
+    long npart = ps.second["position"]["x"].getExtent()[0];
     series.flush();
 
+    //Conversion from Geant4 system of units (http://geant4.web.cern.ch/sites/geant4.web.cern.ch/files/geant4/collaboration/working_groups/electromagnetic/gallery/units/SystemOfUnits.html) to WarpX (SI)
```

Comment:
We should store this info while writing in Geant4 via `setUnitSI()` :)

Reply:
This is a great idea! I will re-order things in my code that uses Geant4. So we can import into WarpX openPMD files from any code. Thanks for telling me @ax3l !!

Reply:
Hooray! :) As discussed in person:
- you can quickly check if the units of the particles look great by reading your file with [openPMD-viewer](https://github.com/openPMD/openPMD-viewer/) in project Jupyter
- when reading here, do [`unitSI()`](https://openpmd-api.readthedocs.io/en/0.11.1-alpha/usage/firstread.html#units) on each record component and multiply the loaded data by its magic value :)

Reply:
Thanks for the help! I will do both things yes and report back on their success :)